src/include/basil.h

```diff
@@ -649,6 +735,13 @@ typedef struct basil_node_memory {
 	struct basil_node_memory *next;
 } basil_node_memory_t;
 
+/* This computeunit struct is added by Altair */
```

Comment:
I can explain why we might state this, when Altair first got basil.h from Cray it contained everything we needed, however, as more BASIL version came out, basil.h didn't always have everything that Altair needed in order to parse/use/store the information, so Altair had to add code to the header file.  And these types of comments help us to know what we added, so that when we move to the next basil.h we'll know to merge this code...but I understand that we shouldn't say the equivalent of "Altair wrote this".  How about if instead of continuing to add to/modify basil.h, we put all the code we'd want to add to basil.h into a separate header file (e.g. pbs_basil.h or some such)? 
There may be other code that has already been added to basil.h, and I'm not suggesting that @dilip-krishnan try to determine all of those...perhaps a bug can be filed to move the existing code that Altair added to basil.h.  But all new additions, like this one, should be added to a new header file.  @mike0042  and @dilip-krishnan what do you think?